packages/documentation/docs/user-guide/installation/installing-a-gateway/input-gateway.md

```
@@ -31,14 +35,18 @@ Currently, input gateway supports:
-Input Gateway supports the concept of _Shift Registers_. A Shift Register is an internal variable/state that can be modified using Actions, from within [Action Triggers](../configuration/settings-view.md#actions). This allows for things such as pagination, _Hold Shift + Another Button_ scenarios, and others on input devices that don't support these features natively. _Shift Registers_ are also global for all devices attached to a single Input Gateway. This allows combining multiple Input devices into a single Control Surface.
+Input Gateway supports the concept of _Shift Registers_. A Shift Register is an internal variable/state that can be modified using Actions, from within [Action Triggers](../../configuration/settings-view.md#actions). This allows for things such as pagination, _Hold Shift + Another Button_ scenarios, and others on input devices that don't support these features natively. _Shift Registers_ are also global for all devices attached to a single Input Gateway. This allows combining multiple Input devices into a single Control Surface.
 
 When one of the _Shift Registers_ is set to a value other than `0` (their default state), all triggers sent from that Input Gateway become prefixed with a serialized state of the state registers, making the combination of a _Shift Registers_ state and a trigger unique.
 
 If you would like to have the same trigger cause the same action in various Shift Register states, add multiple Triggers to the same Action, with different Shift Register combinations.
 
 Input Gateway supports an unlimited number of Shift Registers, Shift Register numbering starts at 0.
 
+### AdLib Tally
+
+Starting with version 0.5.0, Input Gateway can show additional infromation about the playout state of AdLibs. Select device integrations within Input Gateway support *Styles* which allow elements of the HID devices to be specifically styled. These Style classes are mached with [Action Triggers](../../configuration/settings-view.md#action-triggers) using Style class names. You can configure additional *Style classes* when a given AdLib is "active" (currently playing) or "next" (i.e. will be playing after a take) appending a suffix `:active` and `:next` to a Style class name.
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)